wire-ios-sync-engine/Tests/Source/UserSession/SearchTaskTests.swift

```diff
@@ -1161,6 +1161,24 @@ final class SearchTaskTests: DatabaseTest {
         XCTAssertEqual(mockTransportSession.receivedRequests().first?.path, "/users/\(userId.transportString())")
     }
 
+    func testThatItSendsAUserLookupRequest_IfApiVersionIsV2AndAbove() {
+        // given
+        BackendInfo.apiVersion = .v2
+        let userId = UUID()
+        let domain = "wire.com"
+        let task = makeSearchTask(lookupUserId: userId, domain: domain)
+
+        // when
+        task.performUserLookup()
+        XCTAssertTrue(waitForAllGroupsToBeEmpty(withTimeout: 0.5))
+
+        // then
+        XCTAssertEqual(
+            mockTransportSession.receivedRequests().first?.path,
+            "/v3/users/\(domain)/\(userId.transportString())"
```

Comment:
question: Should that be `v2`?

```suggestion
            "/v2/users/\(domain)/\(userId.transportString())"
```